R/visr.R

```diff
@@ -235,11 +235,18 @@ visr.survfit <- function(
 
 # Obtain X-asis label ----------------------------------------------------------
 
-  if (is.null(x_label)){
+  if (is.null(x_label)) {
+    
     if ("PARAM" %in% names(x)) x_label = x[["PARAM"]]
     if (! "PARAM" %in% names(x)) x_label = "time"
     if (!is.null(x_units)) x_label = paste0(x_label, " (", x_units, ")")
```

Comment:
this does not make sense. if the label is not specified, then units should not be added as well. You agree looking at the testing script :)

Reply:
I think it does make sense, as for example in the example below.

```
adtte %>%
  visR::estimate_KM("SEX") %>%
  visR::visr(x_label = NULL,
             x_units = "days")
```
![image](https://user-images.githubusercontent.com/12818242/120994097-7657ad00-c784-11eb-8a03-865e80b0aed7.png)

Reply:
Sure but here the label is extracted from the dataset. When no PARAM is available then it does not make sense.